frontend/index.html

```diff
@@ -183,20 +193,27 @@
         <button type="button" class="custom-toggler site-navbar-toggler site-searchbar-toggle d-none d-md-inline" data-toggle="collapse" data-target="#searchbar-collapse" aria-expanded="false" aria-label="Toggle Search">
           <span class="fa fa-search" aria-hidden="true"></span>
         </button>
+
         <!--Translation Button-->
+        <!--
         <div class="dropdown">
           <button class="dropbtn">Language</button>
-          <div class="dropdown-content">
-            <a href="#" onclick="">English</a>
-            <a href="#" onclick="getSpanishPage()">Español</a>
-            <script >
-              function getSpanishPage(){
-                const STUDY_NAME_HREF = _getStudyName(window.location);
-                window.location.href = `index_es.html?study_config=${STUDY_NAME_HREF}`;
-              }
-            </script>
+          <div class="language dropdown-content">
+            <a href="#" value="en">English</a>
+            <a href="#" value="es">Español</a>
           </div>
         </div>
+      -->
+
+      <!-- Experimental button; It will be removed-->
+
+      <div class="lead">
+        <select id ="buttonBox" class="language" onchange="getLang()">
+          <option value="en">English</option>;
+          <option value="es">Español</option>;         
+        </select>
+      </div>
+
```

Comment:
if this is removed, then how can the language be changed?

Reply:
By "removed" we mean that this button will be replaced with the original one. The one that is orange and green (it can be seen at https://uprm-civic-openpath.nrel.gov/join/).  The one that you are mentioning (it can be seen in the demo video above) is just temporary, unless you prefer this one instead of the original button. Do let us know.